app/src/main/java/org/fossasia/openevent/app/module/event/list/EventListFragment.java

```diff
@@ -137,6 +171,7 @@ public void onStop() {
     private void setupRecyclerView() {
         if (!initialized) {
             eventListAdapter = new EventsListAdapter(getPresenter().getEvents(), bus);
+            eventListAdapter.setSortBy(SORTBYDATE);
```

Comment:
Why are we sorting it initially? Did you try rotating and checking if sort is preserved?